frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -56,6 +56,11 @@ type Props = {| ...OwnProps, ...StateProps |};
 
 type TabKey = "data" | "general" | "defaultConfig";
 
+type DataSourceSettingsStatus = {
```

Comment:
Could you add a small comment indicating what "NONE" means? It becomes apparent that it means "The server has no suggestions." when reading the code, but it's not really clear here in my opinion.

Reply:
I renamed it to `NoAvailableSuggestions` :+1: